thirdparty/cmdconfig/commands/fnruncmd.go

```diff
@@ -0,0 +1,323 @@
+// Copyright 2019 The Kubernetes Authors.
```

Comment:
can you confirm this is a pristine copy of the upstream with no changes?

The first commit should ideally be pristine, so it's easy to figure out what changes have been made as subsequent commits. This should be fine even if the pristine copy doesn't correctly build, it can be fixed later.

Reply:
Updated the PR. The 1st commit contains the pristine copy of upstream.

Reply:
Updated readme. `kyaml` is copied from v0.10.15 and `cmd/config` is from v0.9.7. They are both latest versions.